app/src/services/answers.service.js

```diff
@@ -29,10 +29,10 @@ class AnswerService {
     const { templateid, reportid } = params;
     logger.info(`Getting answer with id ${reportid} of template id ${templateid}`);
     try {
-      const baseURL = config.get("formsAPI.url");
+      const baseURL = config.get("coreAPI.url");
       const response = await axios.default({
         baseURL,
-        url: `/v3/reports/${templateid}/answers/export/${reportid}`,
+        url: `/templates/${templateid}/answers/${reportid}`,
```

Comment:
This endpoint used to allow getting an answer without a template id, this is necessary for the export images as only an answer id is provided in the request